test/standalone/x86_cpu_model/build.zig

```diff
@@ -34,6 +33,17 @@ pub fn build(b: *std.Build) void {
     check_exe.checkContains("__cpu_indicator_init");
     test_step.dependOn(&check_exe.step);
 
+    const run_exe = b.addRunArtifact(exe);
+    if (std.Target.x86.featureSetHas(cpu.features, .avx512vnni)) {
+        run_exe.expectExitCode(3);
+    } else if (std.Target.x86.featureSetHas(cpu.features, .avx2)) {
+        run_exe.expectExitCode(2);
+    } else {
+        run_exe.expectExitCode(1);
+    }
```

Comment:
This'll work for most cases but it isn't quite correct: Multiversioning will dynamically detect the features of the system the program is run on. The current approach breaks if I invoke this test manually with `-Dcpu=...`. So really, you should be checking `b.graph.host` features here.